SCED_localization.py

```
@@ -1909,16 +1880,14 @@
     ensure_dir(args.repo_dir)
     repo_name = repo.split("/")[-1]
     repo_folder = f"{args.repo_dir}/{repo_name}"
-    subprocess.run(
-        ["git", "clone", "--quiet", f"https://github.com/{repo}.git", repo_folder]
-    )
+    subprocess.run(["git", "clone", "--quiet", f"https://github.com/{repo}.git", repo_folder])
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)